src/js/Setup.js

```diff
@@ -5,8 +5,46 @@ var cslib;
 
 var cscompiled = {};
 
+// Simulation settings
 var csanimating = false;
 var csstopped = true;
+var simtime = 0; // accumulated simulation time since start
+var simspeed = 0.5; // global speed setting, will get scaled for applications
+var simcap = 1000 / 20; // max. ms between frames for fps-independent sim
+var simtick = 0; // Date.now of the most recent simulation tick
+var simaccuracy = 10; // number of sub-steps per frame
+
+var simunit = 5 / 360; // reported simulationtime() per internal simtime unit
+/* Cinderella has a factor 5 for its internal animation clock,
+ * and the division by 360 is in the simulationtime function implementation.
+ */
+
+// internal simtime units per millisecond at simspeed 1
+var simfactor = 0.32 / simunit / 1000 * 2;
```

Comment:
This seems to be very hardware dependent. On my machine (Intel(R) Core(TM) i7-4712HQ CPU @ 2.30GHz) a factor of 0.32 is way to high. The swarm for example looks like someone feed them some funny pills.